### ISSUE TITLE
Enable supressedErrors and supressedWarnings in zksolc

### ISSUE THREAD

**Issue by uF4No** [matter-labs/foundry-zksync#633]
### Component

Forge

### Describe the feature you would like

in Hardhat we have this setting in zksolc to disable some errors thrown from the compiler, like when contracts use `transfer` or `tx.origin`:

```
zksolc:{
    settings:{
      suppressedErrors:["sendtransfer", "txorigin"]
    }
  },
```
This is passed to the zksolc compiler json input. 

I have not found any setting in the config to enable this.

### Additional context

Some projects fail to compile without this feature as the compiler sometimes throws these errors as false positives

**Comment by Szczepoo13** [comment]
bump

**Comment by dutterbutter** [comment]
@elfedy can you look to include the argument to `zksolc` to configure suppressing errors / warnings as described [here](https://github.com/matter-labs/era-compiler-solidity/blob/c33431ff06519f0282efb8a43aa859b537241455/era-compiler-solidity/src/zksolc/arguments.rs#L205).

**Comment by dutterbutter** [comment]
@uF4No PR is progress here: https://github.com/Moonsong-Labs/compilers/pull/33

**Comment by elfedy** [comment]
@dutterbutter sure, this will be included in compilers `v0.11.4` so will work on this after the dependency update that includes that version

**Comment by elfedy** [comment]
Closed by: https://github.com/matter-labs/foundry-zksync/pull/704
